lib/platform/azure/index.ts

```diff
@@ -239,6 +244,31 @@ export async function getPr(pullRequestId: number): Promise<Pr | null> {
   return pr;
 }
 
+export async function getPrList(): Promise<Pr[]> {
+  logger.debug('getPrList()');
+  if (!config.prList) {
+    const azureApiGit = await azureApi.gitApi();
+    let prs: GitPullRequest[] = [];
+    let fetchedPrs: GitPullRequest[];
+    let skip = 0;
+    do {
+      fetchedPrs = await azureApiGit.getPullRequests(
+        config.repoId,
+        { status: 4 },
+        config.project,
+        0,
+        skip,
+        100
+      );
+      prs = prs.concat(fetchedPrs);
+      skip += 100;
+    } while (fetchedPrs.length > 0);
```

Comment:
Endless loop in tests, because mock implementation of azure GitApi always returns 1 PR!

https://github.com/renovatebot/renovate/blob/master/test/platform/azure/index.spec.ts#L202-L214
and more below this